### DISCUSSION TITLE
Selecting an Xcode version?

### DISCUSSION THREAD

**Discussion by jshier** [community/community#25870] [Actions]
<p>According to the documentation, the macOS-latest image includes many versions of Xcode. But how do I select one? </p>

**Comment by rectalogic** [comment]
<p>xcode-select -switch</p>

**Comment by scenee** [comment, marked as answer]
<p>As another way, you can use <code>DEVELOPER_DIR</code> env variable.</p>
<pre><code>env:
        DEVELOPER_DIR: /Applications/Xcode_9.4.1.app/Contents/Developer
</code></pre>

**Comment by jshier** [comment]
<p>Thanks. I guess I expected some specific syntax for Xcode version, not to have to set the version manually.</p>
